test/test_bot.py

```diff
@@ -1060,6 +1061,59 @@ def testrule(bot, trigger):
     assert items == [1], 'There must not be any new item'
 
 
+def test_call_rule_pluginabort_with_message(
+    mockbot: bot.Sopel,
+    match_hello_rule: typing.Callable,
```

Comment:
FYI you should not need to annotate test functions. I'm not sure it serves a purpose since these are fixtures.

Reply:
I think I used `test_call_rule()` as my template and took them from there. I don't mind doing it when it indicates the type provided by `pytest` for that fixture but I was just following style from what I copy-pasted :sweat_smile:

Reply:
Yep, I see that some functions have and some don't. Doesn't matter because mypy isn't set to check this directory; just saying you can save some effort if not copy-pasting :)

Reply:
I tend to type annotate in test because it helps my code editor with auto-completion.